collectors/VCenterPropertiesCollector.py

```diff
@@ -0,0 +1,53 @@
+from BaseCollector import BaseCollector
+import os, time
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+
+class VcenterPropertiesCollector(BaseCollector):
+    def __init__(self):
+        self.iteration = 0
+        while not self.iteration:
+            time.sleep(5)
+            self.get_iteration()
+            print("waiting for initial iteration")
+        print("done: initial query")
+        self.property_yaml = YamlRead('collectors/property.yaml').run()
+
+    def collect(self):
+        if os.environ['DEBUG'] >= '1':
+            print('vCenterPropertiesCollector starts with collecting the properties')
+
+        g = GaugeMetricFamily('vrops_vcenter_properties', 'testtest',
+                              labels=['datacenter', 'vcenter', 'propkey'])
+
+        for vc in self.get_vcenters():
+            target = self.vcenters[vc]['target']
+            target_vc = self.vcenters[vc]['name']
+            token = self.vcenters[vc]['token']
+            vc_uuid = self.vcenters[vc]['uuid']
+            for cl in self.get_clusters():
+                if self.clusters[cl]['vcenter'] == target_vc:
+                    dc_name = self.clusters[cl]['parent_dc_name']
```

Comment:
A suggestion for improvement: (If you want the datacenter information in your metric)

```
        for dc in self.get_datacenters():
            target = self.datacenters[dc]['target']
            target_vc = self.datacenters[dc]['parent_vcenter_name']
            token = self.datacenters[dc]['token']
            vc_uuid = self.datacenters[dc]['parent_vcenter_uuid']
            dc_name = self.datacenters[dc]['name']
```


But you don't have to set `dc_name` and `target_vc`. You can insert `self.datacenters[dc]['name']` and `self.datacenters[dc]['parent_vcenter_name']` directly in your labels.